kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSessionManager.scala

```diff
@@ -52,12 +52,8 @@ class KyuubiSessionManager private (name: String) extends SessionManager(name) {
 
   val operationManager = new KyuubiOperationManager()
   val credentialsManager = new HadoopCredentialsManager()
-  val applicationManager = new KyuubiApplicationManager()
-
-  // Currently, the metadata manager is used by the REST frontend which provides batch job APIs,
-  // so we initialize it only when Kyuubi starts with the REST frontend.
-  lazy val metadataManager: Option[MetadataManager] =
-    if (conf.isRESTEnabled) Some(new MetadataManager()) else None
+  val metadataManager: MetadataManager = new MetadataManager()
```

Comment:
It is fine to always create metadatamanager, especially we are going to persist the kubernetes information into database.